src/pages/app.tsx

```diff
@@ -171,14 +171,14 @@ const PlatformsInstallSnippet: FunctionComponent = () => (
 
                                     <CodeSnippet
                                         code="brew install sourcegraph/sourcegraph-app/sourcegraph"
-                                        className="-ml-2.5 pb-3"
+                                        className="-ml-2.5"
                                     />
                                 </li>
-                                <li className="ml-2.5 text-xs leading-7">Run `Sourcegraph` in your local directory.</li>
-                                <li className="ml-2.5 text-xs leading-7">
+                                <li className="ml-2.5 text-sm leading-7">Run `Sourcegraph` in your local directory.</li>
```

Comment:
Should this be

```suggestion
                                <li className="ml-2.5 text-sm leading-7">Run <code>sourcegraph</code> in your local directory.</li>
```

(If we have `<code>` styled already, that is).

Should definitely be lowercased, though!

Reply:
Not by default, but with a few styles we can get something similar:

`<code className="bg-gray-200 text-xs rounded-md px-2 text-gray-700">sourcegraph</code>`

<img width="731" alt="image" src="https://user-images.githubusercontent.com/9516420/224730646-fe4207dd-97b9-4c49-92b6-12a6604a3540.png">

WDYT?